src/main/java/seedu/sellsavvy/model/person/Person.java

```diff
@@ -156,4 +156,17 @@ public String toString() {
                 .toString();
     }
 
+    /**
+     * Creates a new copy of this {@code Person} with new attributes.
+     * Only the tags are not copied.
+     */
+    public Person createCopy() {
+        Name name = new Name(getName().toString());
+        Address address = new Address(getAddress().toString());
+        Phone phone = new Phone(getPhone().toString());
+        Email email = new Email(getEmail().toString());
+        Set<Tag> tags = getTags();
+        OrderList orderList = getOrderList().createCopy();
```

Comment:
Can u make the predicate same as well?

Reply:
there is no predicate field for Person but i'll copy over from filteredOrders and apply it to the new created Person object